templates/terraform/environments/prod/main.tf

```diff
@@ -53,4 +53,8 @@ module "prod" {
   <% if ne (index .Params `loggingType`) "kibana" %># <% end %>logging_es_instance_count = "2" # Must be a mulitple of the az count
   <% if ne (index .Params `loggingType`) "kibana" %># <% end %>logging_volume_size_in_gb = "50" # Maximum value is limited by the instance type
   # See https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/aes-limits.html
+
+  sendgrid_enabled = var.sendgrid_enabled
+  sendgrid_cname = var.sendgrid_cname
```

Comment:
This should be templated rather than a var?

Reply:
I'm relying on the script to generate tfvars.json that will look something like 
```js
//terraform/environments/stage/sendgrid.auto.tfvars.json
{
  "sendgrid_enabled": true,
  "sendgrid_domain_id": 9104854,
  "sendgrid_cname": [
    [
      "em7596.email.get0.dev",
      "u18291290.wl041.sendgrid.net"
    ],
    [
      "s1._domainkey.email.get0.dev",
      "s1.domainkey.u18291290.wl041.sendgrid.net"
    ],
    [
      "s2._domainkey.email.get0.dev",
      "s2.domainkey.u18291290.wl041.sendgrid.net"
    ]
  ]
}

// terraform/environments/stage/sendgrid-secrets.auto.tfvars.json
{"sendgrid_apikey": "SG.blahblah-secret"}

```

Reply:
Oh yeah, I must have missed that. Hm, not sure how I feel about this.. I'll think a bit more about it.

Reply:
Yeah, actually this would make the tf totally unusable.

Reply:
with the variables and default values in the stage/prod, they will just go to the default (enabled:false) and empty values